R/derive_var_dthcaus.R

```diff
@@ -78,33 +81,51 @@
 #' )
 #'
 #' src_ds <- dthcaus_source(
-#'   dataset = ds,
+#'   dataset_name = "ds",
 #'   filter = DSDECOD == "DEATH" & grepl("DEATH DUE TO", DSTERM),
 #'   date = DSSTDTC,
 #'   mode = "first",
 #'   dthcaus = DSTERM,
 #'   traceability_vars = vars(DTHDOM = "DS", DTHSEQ = DSSEQ)
 #' )
 #'
-#' derive_var_dthcaus(adsl, src_ae, src_ds)
+#' derive_var_dthcaus(adsl, src_ae, src_ds, source_datasets = list(ae = ae, ds = ds))
 derive_var_dthcaus <- function(dataset,
                                ...,
+                               source_datasets,
                                subject_keys = vars(STUDYID, USUBJID)) {
-
-  assert_data_frame(dataset, subject_keys)
+  assert_data_frame(dataset)
```

Comment:
Why has the check on required variables (`subject_keys`) been removed?

Reply:
That was not intentional.